cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -34,7 +34,7 @@ include::modules/technology-preview.adoc[leveloffset=+2]
 
 // CNV-4076 UI view/config dedicated resources
 
-// CNV-4077 Improved UI template validation
+* The web-console is now capable of validating CPU and Memory sizes and disk bus requirements of different operating systems. This enables the web console to raise resource warnings prior to virtual machine creation. 
```

Comment:
- s/web-console/web console
- s/Memory/memory
- This is sort of vague and anthropomorphizes the web console a bit :) When do these resource warnings get raised? During the VM wizard process? 
- I'd rephrase it to be something like: "The VM creation wizard now validates CPU, memory, and disk bus requirements for different operating systems (side note- which OSes?). If you attempt to create a VM that does not meet OS requirements, the wizard now raises resource warnings."